packages/python-future/build.sh

```diff
@@ -0,0 +1,27 @@
+TERMUX_PKG_HOMEPAGE=https://python-future.org
+TERMUX_PKG_DESCRIPTION="Clean single-source support for Python 3 and 2"
+TERMUX_PKG_LICENSE="MIT"
+TERMUX_PKG_MAINTAINER="@fervi"
+TERMUX_PKG_VERSION=0.18.2
+TERMUX_PKG_SRCURL=https://github.com/PythonCharmers/python-future/archive/refs/tags/v${TERMUX_PKG_VERSION}.tar.gz
+TERMUX_PKG_SHA256=43c1feae4170742671ffef900acd5dbe7c72099aa602d58e95e22c2174edd057
+TERMUX_PKG_DEPENDS="python"
+TERMUX_PKG_BUILD_IN_SRC=true
+
+_PYTHON_VERSION=$(. $TERMUX_SCRIPTDIR/packages/python/build.sh; echo $_MAJOR_VERSION)
```

Comment:
Please move this into `termux_step_pre_configure`. We prefer to have the build recipes be sourceable without issues, and having it here gives:

```sh
$ source packages/python-future/build.sh 
bash: /packages/python/build.sh: No such file or directory
```

Since TERMUX_SCRIPTDIR is not defined in the recipe